api/comments_api.py

```diff
@@ -39,10 +42,22 @@ class CommentsAPI(basehandlers.APIHandler):
   """Users may see the list of comments on one of the approvals of a feature,
    and add their own, if allowed."""
 
+  def _should_show_comment(self, comment, email, is_admin):
+    # If the comment is deleted and the current user can't view it,
```

Comment:
This can be a docstring rather than a comment.  Ideally it would be one line, e.g.:
`"""Only send comments that the user can view."""`

Reply:
I changed up the wording a bit that is hopefully sufficient.

`"""Check whether a comment should be visible to the user."""`